packages/ckeditor5-horizontal-line/docs/features/horizontal-line.md

```diff
@@ -5,14 +5,19 @@ menu-title: Horizontal line
 
 # Horizontal line
 
-The {@link module:horizontal-line/horizontalline~HorizontalLine} plugin provides the possibility to insert a horizontal line into the rich-text editor.
+The {@link module:horizontal-line/horizontalline~HorizontalLine} plugin provides the possibility to insert a horizontal line into the rich-text editor. Often known as the horizontal rule, this provides a visual way to separate the content. It may indicate and thematic shift (like change of topic) between paragraphs or parts of the story or just be a simple graphic separator to help organize the content.
```

Comment:
```suggestion
The {@link module:horizontal-line/horizontalline~HorizontalLine} plugin provides the possibility to insert a horizontal line into the rich-text editor. Often known as the horizontal rule, this provides a visual way to separate the content. It may indicate a thematic shift (like change of topic) between paragraphs or parts of the story or just be a simple graphic separator to help organize the content.
```